includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -2498,6 +2504,20 @@ private function get_appearance_transient_key( $is_block_checkout = false ) {
 		return ( $is_block_checkout ? self::BLOCKS_APPEARANCE_TRANSIENT : self::APPEARANCE_TRANSIENT ) . '_' . get_option( 'stylesheet' );
 	}
 
+	/**
+	 * Checks if the current page is the order details page.
+	 *
+	 * @return bool Whether the current page is the order details page.
+	 */
+	private function is_order_details_page() {
+		$query_params = wp_unslash( $_GET ); // phpcs:ignore WordPress.Security.NonceVerification.Missing
+		if ( function_exists( 'wcs_is_custom_order_tables_usage_enabled' ) && wcs_is_custom_order_tables_usage_enabled() ) { // If custom order tables are enabled, we need to check the page query param.
```

Comment:
`wcs_is_custom_order_tables_usage_enabled` is a subscriptions function so won't always be available.

The Stripe way of checking if custom tables are enabled is this:

```
 class_exists( 'Automattic\WooCommerce\Utilities\OrderUtil' ) && OrderUtil::custom_orders_table_usage_is_enabled()
 ``` 

There's already a few uses of it so probably worth making our own helper/compatibility function for it.

Reply:
Oops, I didn't realize that. I created a new helper class with a method for this and replaced all calls to use it instead 👍